handlers/me.go

```diff
@@ -0,0 +1,42 @@
+package handlers
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/gin-gonic/gin"
+	database "speedrun.website/db"
+	"speedrun.website/graph/model"
+	"speedrun.website/middleware"
+)
+
+func MeHandler(c *gin.Context) {
+	user, _ := c.Get(middleware.JwtConfig.IdentityKey)
+	db, err := database.GetDatabase()
```

Comment:
We may want to be using golang connection pooling a bit more, but for "getting it running and functional" and for MVP purposes, this should be good.

Eventually this may cause some performance issues (either inside the code, or on the DB side from connection bloat or constantly opening/closing via network traffic) later and could potentially use a DB connection paradigm rework.

Reply:
I'm not finding some clear answers online but it seems that we could call `gorm.Open` only once and use the same db afterward but I'm not entirely sure so I'll leave as it for now if that's ok with everyone.

Reply:
Yeah, there are some patterns. I think we should leave this be as now, more of a comment for future thought.